proposals/0024-opacity-micromaps.md

```diff
@@ -78,10 +78,13 @@ doesn't apply for inline raytracing however. `RayQuery` objects are
 independent of raytracing pipelines. For `RayQuery` the template for 
 instantiating the object includes a new optional `RAYQUERY_FLAGS` parameter:
 
-`RayQuery<RAY_FLAGS, RAYQUERY_FLAGS>`
+```
+template<uint StaticRayFlags, uint RayQueryFlags = RAYQUERY_FLAG_NONE>
+class RayQuery;
+```
 
 ```hlsl
-enum RAYQUERY_FLAG : uint
+enum class RAYQUERY_FLAG : uint32_t
```

Comment:
Should probably back-out this changed line.

This is HSL-equivalent code, which doesn't actually have a defined enum type, so it represents a kind of virtual ("as-if") enum.  The defined values are not scoped under an enum, since they are actually `static const uint` values, so enum "class" is inappropriate, as that would change the scope of the values.

Also, since this is HLSL, `uint` is fine as the storage type as well.